BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 The Dawn Authors
```

Comment:
I am not a lawyer so I'm not sure if this is a problem, but the Copyright statement might need modification because this file is from the Dawn Authors until it was modified for WebNN at which point it both the Dawn Authors and the WebNN-native Authors.

Recording the difference might be useful to know what files have modifications not included in the Google CLA is this ever gets merged into Dawn. If the merge happens there will be other issues around copyrights because historical contributions by everyone will need to go under the Google CLA and how to do that is ???

Reply:
> both the Dawn Authors and the WebNN-native Authors.

Good point, probably we need to add `Copyright 2021 WebNN-native authors` for the modifications. @anssiko , please comment.

> Recording the difference might be useful

Agree. For all the files derived from Dawn, the Dawn Authors Copyright statement is kept. There are 44 files in this PR. And there is git commit id of Dawn in [DEPS](https://github.com/webmachinelearning/webnn-native/blob/5483049d6aa356b72961587e84eff20ac95e1f2f/DEPS#L15). It records where we started. Diff these files against ones of Dawn at that point will show the modifications. Is this good?

Reply:
@Kangz thanks for this comment. We'll look into this detail. I suggest we keep this PR open until this question has been addressed.

Reply:
It's probably ok to know the base commit and add `WebNN-native authors` if there are modifications but someone's legal team should confirm that.

Reply:
@Kangz we are working with our legal team now to clarify this particular detail, and will provide an update in this PR.

Reply:
After confirm with our legal team, it is OK to add `The WebNN-native Authors` for the modification. The new commit https://github.com/webmachinelearning/webnn-native/pull/1/commits/beb8ad99af24f11b33c6306a903c640d647d75ca contains this change. For the record, in this PR, there are 47 files derived from Dawn project at [commit](https://dawn.googlesource.com/dawn/+/bf1c0cf52377b4db2bf3a433dc5056620aad7cdd). For these derived files, they keep the original The Dawn Authors copyright header and append The WebNN-native Authors copyright header for the modifications. 

@Kangz and @anssiko, please take another look. Thanks.

Reply:
LGTM